python/examples/11-mailbox-agents/bob.py

```diff
@@ -11,18 +11,12 @@ class Message(Model):
 # Generate a second seed phrase (e.g. https://pypi.org/project/mnemonic/)
 SEED_PHRASE = "put_your_seed_phrase_here"
 
-# Copy the address shown below
-print(f"Your agent's address is: {Agent(seed=SEED_PHRASE).address}")
-
-# Then go to https://agentverse.ai, register your agent in the Mailroom
-# and copy the agent's mailbox key
-AGENT_MAILBOX_KEY = "put_your_AGENT_MAILBOX_KEY_here"
-
 # Now your agent is ready to join the agentverse!
```

Comment:
Since we're not configuring anything that's called "mailbox" it is less obvious to see what kind of agent is being configured, if you don't already know. Also if I understand correctly, the only way now we're officially supporting is to use the local inspector (?). So I'd suggest to add more commentary too guide the user:

\# By specifying the agentverse submit endpoint, your agent will automatically know it's a mailbox agent. In order to complete registration, you need to visit your agents local inspector interface and connect it with your agentverse account.

We should add or link to more documentation regarding this process as soon as it is available.